### DISCUSSION TITLE
`brew bump` for a prerelease

### DISCUSSION THREAD

**Discussion by toobuntu** [Homebrew/discussions#5229] [Tap maintenance and Homebrew development] [answered]
### Output of `brew config`

```shell
N/A
```

### Output of `brew doctor`

```shell
N/A
```

### Description of issue

Is it possible to `brew bump` when the cask comes from a GitHub release labeled as prerelease? If not, is it possible to exclude just the prerelease audit check in `brew bump` or `brew bump-cask-pr`? All releases of freetube have so far been labeled prerelease (whether in homebrew-cask or a third-party tap).

```bash
brew tap pikachuexe/freetube
HOMEBREW_DEVELOPER=1 HOMEBREW_GITHUB_API_TOKEN="$(gh auth token)" brew bump --open-pr --tap pikachuexe/freetube --casks pikachuexe/freetube/pikachuexe-freetube
...
audit for pikachuexe-freetube: failed
 - v0.19.2-beta is a GitHub pre-release.
pikachuexe/freetube/pikachuexe-freetube
  * line 5, col 2: v0.19.2-beta is a GitHub pre-release.
Error: 1 problem in 1 cask detected.
Error: `brew audit` failed!
```

**Comment by Bo98** [comment]
In Homebrew/cask we do this: https://github.com/Homebrew/homebrew-cask/blob/66b3638da27ec3aa763f20d679e21bb0f9f2fe8f/audit_exceptions/github_prerelease_allowlist.json#L10

**Reply by toobuntu** [reply]
Thank you. Is it possible to locally (actually, in a GitHub Actions workflow) add the cask from this third-party tap to this file (using the fully qualified name, for example) so it can apply when running brew bump?

**Reply by Bo98** [reply, marked as answer]
It's a per-tap thing so you can add your own `audit_exceptions/github_prerelease_allowlist.json` file to your tap and it should work fine. Given it checks within the same tap I don't think it checks the fully-qualified name so just the basename should work.
